.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.5
```

Comment:
Ooh what is this??

Reply:
This is for `asdf` and other tools that manage versions for different languages

Reply:
Mange Is to manage other languages

Reply:
I removed `.tool-version` as it's kind of redundant. Multiple language manages can fallback to ` .ruby-version `.

Reply:
Ok! No worries. I was just wondering! Regardless `asdf` seems like a pretty cool tool — I don't know how I've never heard of it before 🤯